README.md

```diff
@@ -33,7 +33,7 @@ Clone the git repository:
 	git clone https://github.com/RaymiiOrg/bash-http-monitoring.git
 	cd bash-http-monitoring
 
-Edit the `srvmon.sh` script and add your sites. A few examples are provided. This is the syntax:
```

Comment:
I'm not fond of removing the extension, could you please explain why that would be necessary for this request?

Reply:
Mostly due to the configuration of the docker image I've picked:

https://github.com/hypoport/httpd-cgi/blob/master/httpd.conf#L71

Reply:
Clear enough, thanks for clarifying